components/resources/library/src/commonTest/kotlin/org/jetbrains/compose/resources/TestUtils.kt

```diff
@@ -66,4 +69,16 @@ internal fun parsePluralSamples(samples: String): List<Int> {
 internal fun pluralRuleListOf(vararg rules: Pair<PluralCategory, String>): PluralRuleList {
     val pluralRules = rules.map { PluralRule(it.first, it.second) } + PluralRule(PluralCategory.OTHER, "")
     return PluralRuleList(pluralRules.toTypedArray())
-}
\ No newline at end of file
+}
+
+/**
+ * Executes a test block within a Compose UI testing environment while ensuring
+ * that any cached resources are cleared before the test begins.
+ */
+@OptIn(ExperimentalTestApi::class)
+fun runComposeResourceTest(block: suspend ComposeUiTest.() -> Unit) = runComposeUiTest {
```

Comment:
I'd name it `clearResourceCachesAndRunUiTest`. this is much easier for understanding

Reply:
done